lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -263,3 +246,59 @@ function dosomething_rogue_rb_exists_in_rogue($rbid)
 {
   return db_query("SELECT * FROM {dosomething_rogue_reportbacks} rogue_rbs WHERE rbid = :rbid", array(':rbid' => $rbid))->fetchAll();
 }
+
+/**
+ * Transform reportback into the appropriate scheme to send to Rogue.
+ *
+ * @param $northstar_id - User's Northstar user id.
```

Comment:
you should probably only send one user value in, either ns id, or drupal user. I know we are starting to prefer NS id, so it's probably safe to only send that.